app/views/posts/_new.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= form_with model: Post.new, url: '/posts/create', class: 'new-post-form' do |form| %>
+  <%= form.hidden_field :author_id, value: current_user.id %>
+  <%= form.text_field :title, placeholder: 'Title' %>
+  <br>
+  <%= form.text_area :text, placeholder: 'Text' %>
+  <br>
+  <%= form.submit %>
+<% end %>
```

Comment:
- [ ] **[OPTIONAL]** Kindly make this form visible on the page, you can add a little CSS to it.

Reply:
I will add styles later